### ISSUE TITLE
mingo output incorrectly aligned

### ISSUE THREAD

**Issue by simgar** [rust-embedded/rust-raspberrypi-OS-tutorials#229]
I've been using [scip](https://github.com/BartMassey-upstream/scip) with my RPi3 and 'RPi serial output to USB cable'.  I've also been using Python's `miniterm` for comparison.  With the command:

    sudo python3 -m serial.tools.miniterm /dev/ttyUSB0 921600

I get

	--- Miniterm on /dev/ttyUSB0  921600,8,N,1 ---
	--- Quit: Ctrl+] | Menu: Ctrl+T | Help: Ctrl+T followed by Ctrl+H ---
	[0] mingo version 0.5.0
	[1] Booting on: Raspberry Pi 3
	[2] Drivers loaded:
		  1. BCM PL011 UART
		  2. BCM GPIO
	[3] Chars written: 117
	[4] Echoing input now

When I use `scip` with: 

    scip /dev/ttyUSB0 921600

I get:

	Welcome to scip.
	To exit type <Enter> + ~ + .
	or unplug the serial port.
	[0] mingo version 0.5.0
						   [1] Booting on: Raspberry Pi 3
														 [2] Drivers loaded:
																				  1. BCM PL011 UART
						 2. BCM GPIO
									[3] Chars written: 117
														  [4] Echoing input now

[Here](https://github.com/BartMassey-upstream/scip/issues/1), it was stated that `mingo` is only sending `\n` where it should be sending `\r\n`.

**Comment by BartMassey** [comment]
Yes, thank you for the issue report. We are working on fixing this and other things.
